GOT-StudentVersion/GOT-StudentVersion/Controllers/GOTViewController.swift

```diff
@@ -0,0 +1,211 @@
+        return EpisodeSearchResults[section].count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let currentEpisode: GOTEpisode = EpisodeSearchResults[indexPath.section][indexPath.row]
+        
+        switch indexPath.section % 2 {
+        case 0:
+            if let cell = tableView.dequeueReusableCell(withIdentifier: "myGOTCell", for: indexPath) as? GoTEvenNumberCell {
+                cell.episodeNumberLabel.text = "S:\(currentEpisode.season) E:\(currentEpisode.number)"
+                cell.episodePreviewImage.image = currentEpisode.getImage()
+                
+                let attrString = NSAttributedString(string: currentEpisode.name, attributes: [NSAttributedString.Key.strokeColor: UIColor.white, NSAttributedString.Key.foregroundColor: UIColor.yellow, NSAttributedString.Key.strokeWidth: -4.0, NSAttributedString.Key.font: UIFont.systemFont(ofSize: 17.5)])
+                cell.episodeNameLabel.attributedText = attrString
+                
+                let selectedView = UIView()
+                selectedView.backgroundColor = UIColor(red: 0/255, green: 0/255, blue: 0/255, alpha: 1.0)
+                cell.selectedBackgroundView = selectedView
+                return cell
+            }
+            
+        case 1:
+            if let cell = tableView.dequeueReusableCell(withIdentifier: "myOddGOTCell", for: indexPath) as? GoTOddNumberCell {
+                cell.episodeNumberLabel.text = "S:\(currentEpisode.season) E:\(currentEpisode.number)"
+                cell.episodePreviewImage.image = currentEpisode.getImage()
+                
+                let attrString = NSAttributedString(string: currentEpisode.name, attributes: [NSAttributedString.Key.strokeColor: UIColor.white, NSAttributedString.Key.foregroundColor: UIColor.yellow, NSAttributedString.Key.strokeWidth: -4.0, NSAttributedString.Key.font: UIFont.systemFont(ofSize: 17.5)])
+                cell.episodeNameLabel.attributedText = attrString
+                
+                let selectedView = UIView()
+                selectedView.backgroundColor = UIColor(red: 0/255, green: 0/255, blue: 0/255, alpha: 1.0)
+                cell.selectedBackgroundView = selectedView
+                return cell
+            }
+            
+        default: break
+        }
+        
+        return UITableViewCell()
+    }
+}
+
+
+//MARK: -- Table Delegate Methods
+extension GOTViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int){
+        view.tintColor = UIColor(displayP3Red: 0.351, green: 0.121, blue: 0.097, alpha: 1)
+        let header = view as! UITableViewHeaderFooterView
+        header.textLabel?.textColor = UIColor.white
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 115
+    }
+}
+
+
```

Comment:
Great job separating out the search bar functionality with an extension. It really cleans up the VC